File: Data Structures & Algorithms/non-cyclical-number/submission-0.py
class Solution:
    def isHappy(self, n: int) -> bool:
        seen = set()
        while n > 0:
            if n == 1:
                return True
            if n in seen:
                return False
            seen.add(n)
            n = self.squares(n)

    def squares(self, n):
            sum = 0
            for c in str(n):
                sum += int(c) ** 2
            return sum

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int low = 0;
        int high = numbers.length-1;
        int[] indices = new int[2];
        while (low < high){
            int sum = numbers[low] + numbers[high];
            if (sum == target){
                indices[0] = low + 1;
                indices[1] = high + 1;
                return indices;
            }
            else if(sum < target){
                low++;
            }
            else{
                high--;
            }

        }
        return new int[0];
    }
}
